strapi/config/plugins.ts

```diff
@@ -1,6 +1,6 @@
-import meilisearchConfig from "./plugins.meilisearch.config";
+import meilisearchConfig from "./plugins.meilisearch.config"
 
-export default {
+module.exports = ({ env }) => ({
```

Comment:
Is there a reason for changing `export default` to `module.exports`? `export default` is used usually for ths files and ESM